examples/operations/public/js/user-accounts.js

```diff
@@ -1,171 +1,172 @@
     var data = {
-      token: $( '#reset_token' ).val(),
-      password: $( '#new_password' ).val()
+      token: $('#reset_token').val(),
+      password: $('#new_password').val()
     }
-    $.post( '/auth/execute_reset', data, function ( instance ) {
+    $.post('/auth/execute_reset', data, function (instance) {
       cleanMessages()
-      if ( !instance.ok ) {
-        $( '#errorMsg' ).text( 'Error: ' + instance.why )
+      if (!instance.ok) {
+        $('#errorMsg').text('Error: ' + instance.why)
         return
       }
       else {
-        $( '#msg' ).text( 'Password updated successfully.' )
+        $('#msg').text('Password updated successfully.')
       }
       showLogin()
-    } )
+    })
     return false
-  } )
+  })
 
-  $( '#load_reset' ).click( function () {
+  $('#load_reset').click(function () {
     var data = {
-      token: $( '#reset_token' ).val()
+      token: $('#reset_token').val()
     }
-    $.post( '/auth/load_reset', data, function ( instance ) {
+    $.post('/auth/load_reset', data, function (instance) {
       cleanMessages()
-      if ( !instance.ok ) {
-        $( '#errorMsg' ).text( 'Error: ' + instance.why )
+      if (!instance.ok) {
+        $('#errorMsg').text('Error: ' + instance.why)
         return
       }
       else {
-        $( '#msg' ).text( 'Reset token ' + ( instance.active ? 'active' : 'not active' ) + '. User: ' + instance.nick )
+        $('#msg').text('Reset token ' + (instance.active ? 'active' : 'not active') + '. User: ' + instance.nick)
       }
-    } )
+    })
     return false
-  } )
+  })
 
-  $( '#logout' ).click( function () {
+  $('#logout').click(function () {
     cleanMessages()
-    $.post( '/auth/logout', {}, function ( instance ) {
+    $.post('/auth/logout', {}, function (instance) {
       cleanMessages()
-      $( '#msg' ).text( 'Logout successfully.' )
-      showLogin( instance )
-    } )
-  } )
+      $('#msg').text('Logout successfully.')
+      showLogin(instance)
+    })
+  })
 
-  $( '#create_reset' ).click( function () {
+  $('#create_reset').click(function () {
     cleanMessages()
-    $.post( '/auth/create_reset', {nick: $( '#login_username' ).val() }, function ( instance ) {
+    $.post('/auth/create_reset', {nick: $('#login_username').val()}, function (instance) {
       cleanMessages()
-      if ( !instance ) {
-        $( '#errorMsg' ).text( 'Reset password token error. Reason: Unknown reason' )
+      if (!instance) {
+        $('#errorMsg').text('Reset password token error. Reason: Unknown reason')
         return
       }
-      if ( instance.ok ) {
-        $( '#msg' ).text( 'Reset password token created successfully. The token should be sent on owner e-mail. For simplicity we are displaying the token here: ' + instance.reset.id )
+      if (instance.ok) {
+        $('#msg').text('Reset password token created successfully. The token should be sent on owner e-mail. For simplicity we are displaying the token here: ' + instance.reset.id)
```

Comment:
@mihaidma

This will not work if we are not sending the token anymore, we need to fix it somehow. See my comment on cmd_create_reset action.

For this example we should add our own create_reset action - that sends to token in the response.